test/Operators/hybrid/2d.jl

```diff
@@ -547,6 +547,56 @@ end
     @test norm(curluw_diff) < norm(curluw_ref) * 1e-2
 end
 
+@testset "curl (field b.c.)" begin
+    hv_center_space, hv_face_space =
+        hvspace_2D(xlim = (-pi, pi), zlim = (-pi, pi))
+
+    ccoords = Fields.coordinate_field(hv_center_space)
+    fcoords = Fields.coordinate_field(hv_face_space)
+    fcoords_1 = Fields.level(fcoords, ClimaCore.Utilities.half)
+    curl_bcfield = Geometry.Covariant2Vector.(0.0 .* fcoords_1)
```

Comment:
You need to specify a component of `fcoords_1` (`x` or `z`)?
```suggestion
    curl_bcfield = Geometry.Covariant2Vector.(0.0 .* fcoords_1.x)
```